constants.go

```diff
@@ -89,6 +89,7 @@ const (
 	LIST     = "LIST"
 	LUSERS   = "LUSERS"
 	MODE     = "MODE"
+	MONITOR  = "MONITOR"
```

Comment:
This should be placed in the `IRCv3 commands and extensions` section.

Reply:
Fixed